upload/admin/view/template/extension/module/smaily_for_opencart.tpl

```diff
@@ -126,15 +126,39 @@
                 <p><?php echo $rss_feed_text ?></p>
               </div>
             </div>
-            <?php if(!$validated) : ?>
+            <?php if($validated) : ?>
```

Comment:
Reset button logic when validating credentials does not work very well.

When user validates the credentials, then the "Validate" button just disappears, but when user resets credentials then the "Reset" button does not.

I think it should work in the lines of - if user validates the credentials then "Validate" button is replaced with "Reset" and vice versa when user resets credentials.

Reply:
I refactored the template to switch sections based on current state of validation. https://github.com/sendsmaily/smaily-opencart-module/pull/59/commits/4402ab26d87f5f194d060d36eef8c21b0bbb3a04